build_runner/test/daemon/daemon_test.dart

```diff
@@ -47,10 +47,10 @@ void main() {
         ],
       ),
       d.dir('test', [
-        d.file('hello_test.dart', '''
-import 'package:test/test.dart';
+        d.file('hello.dart', '''
 main() {
-  test('hello', () {});
+  // Don't actually use test package to speed up tests
```

Comment:
Can you expand the commit message when you merge to add some detail about this?

Reply:
sure